docker/build/suzuka-full-node-setup/Dockerfile

```diff
@@ -34,20 +38,28 @@ RUN rust_binary="./target/release/suzuka-full-node-setup"; dest_dir="/tmp/runtim
     grep '^/' | xargs -I {} dirname {} | sort | uniq | xargs -I {} \
     bash -c 'mkdir -p "$0/$1" && rsync -a --copy-links "$1/" "$0/$1/"' "$dest_dir" {}
 
+RUN rust_binary="/tmp/executables/anvil"; dest_dir="/tmp/anvil-runtime"; \
```

Comment:
Why not copy these libraries into the same `/tmp/runtime` directory as the previous step does, and save yourself another copy layer in the image? If you say "better incremental caching", then perhaps put this one first because Anvil is going to be changed less often?